src/libraries/System.Private.CoreLib/src/System/Numerics/BitOperations.cs

```diff
@@ -41,6 +41,42 @@ public static class BitOperations
             19, 27, 23, 06, 26, 05, 04, 31
         };
 
+        private static readonly uint[] s_crcTable = new uint[]
```

Comment:
you can reuse the existing table generator with non-revered polynomial value `0x82F63B78u`
https://github.com/kasperk81/runtime/commit/d74d96a9002e6691ef8d501c80e6cd7669db5097

Reply:
@kasperk81 Build failures `Type 'Crc32ReflectedTable' is a static holder type but is neither static nor NotInheritable`

Reply:
add static to the class? you don't had to revert the whole thing.. take your time and build and test it locally